build.gradle

```diff
@@ -118,7 +118,11 @@ configurations {
 }
 minecraft.runs.all {
     lazyToken('minecraft_classpath') {
-        configurations.library.copyRecursive().resolve().collect { it.absolutePath }.join(File.pathSeparator)
+        configurations.library.copyRecursive().resolve()
```

Comment:
Hmm, I think that `minecraftLibrary` is the intended way to do this with newer ForgeGradle versions (in which case, these will need to be excluded in the dependency declaration) - can we move this there already, even if not upgrading FG yet?